Doc/c-api/unicode.rst

```diff
@@ -1423,6 +1423,24 @@ They all return ``NULL`` or ``-1`` if an exception occurs.
    This function returns ``-1`` upon failure, so one should call
    :c:func:`PyErr_Occurred` to check for errors.
 
+   See also :c:func:`PyUnicode_Equal`.
+
+
+.. c:function:: int PyUnicode_Equal(PyObject *a, PyObject *b)
+
+   Test if two strings are equal:
+
+   * Return ``1`` if *a* is equal to *b*.
+   * Return ``0`` if *a* is not equal to *b*.
+   * Set a :exc:`TypeError` exception and return ``-1`` if *a* or *b* is not a
+     Python :class:`str` object.
+
+   The function always succeed if *a* and *b* are Python :class:`str` objects.
```

Comment:
I don't think we need to explicitly mention that `str` is a Python object, the reader should (hopefully) know that already :)
```suggestion
   * Set a :exc:`TypeError` exception and return ``-1`` if *a* or *b* is not a
     :class:`str` object.

   The function always succeeds if *a* and *b* are :class:`str` objects.
```